src/i18n/loadTranslations.js

```diff
@@ -1,4 +1,4 @@
-// AUTO-GENERATED. See src/iNaturalistReactNative/src/i18n/i18ncli.js
+// AUTO-GENERATED. See src/i18n/i18ncli.js
```

Comment:
fwiw i don't know where this is coming from, but i've seen this updating in other PRs as well. not something i intended to change here.